### ISSUE TITLE
can I oxidize a project without it being in pip?

### ISSUE THREAD

**Issue by djotaku** [indygreg/PyOxidizer#108]
It's possible I misread your documentation, but it appears to focus around oxidizing packages in pip. If I just want to oxidize my code (which is not in pip), how do I do that?

**Comment by gnprice** [comment]
Yes, definitely. This ends up being nicely simple once you find it.

Here's one small example: https://github.com/indygreg/PyOxidizer/issues/78#issuecomment-511657412
and here's another: https://github.com/indygreg/PyOxidizer/issues/27#issuecomment-506298186

I agree it'd be good to work this more into the docs.

**Comment by gnprice** [comment]
> I agree it'd be good to work this more into the docs.

(#49 is approximately this suggestion too.)
